docs/docs/configuration/providers/github.md

```diff
@@ -14,30 +14,42 @@ org and team or collaborators.
 
 To restrict by organization only, include the following flag:
 
-    -github-org="": restrict logins to members of this organisation
+```
+    --github-org="": restrict logins to members of this organisation
```

Comment:
Maybe:

```suggestion
```shell
    --github-org=""  # restrict logins to members of this organisation
```